frontend/src/pages/ActivityReport/Pages/Review/Submitter/Draft.js

```diff
@@ -77,7 +78,10 @@ const Draft = ({
   };
 
   const onSubmit = (e) => {
-    if (allGoalsHavePromptResponses && !hasIncompletePages && !grantsMissingMonitoring.length) {
+    if (allGoalsHavePromptResponses
+      && !hasIncompletePages
+      && !grantsMissingMonitoring.length
+      && !grantsMissingCitations.length) {
```

Comment:
might be nice to move this into a variable outside of this function, like 

```js
const canSubmitReport = allGoalsHavePromptResponses
      && !hasIncompletePages
      && !grantsMissingMonitoring.length
      && !grantsMissingCitations.length;
      
const onSubmit = (e) => {
    if (canSubmitReport) {

Reply:
(totally optional)

Reply:
updated.